### DISCUSSION TITLE
sorry can someone help. Sent GO token to from exchange account to ledger X cold wallet, can't find them now.

### DISCUSSION THREAD

**Discussion by SPART20** [gochain-io/docs#110] [Q&A]

**Comment by treeder** [comment]
Check your address on https://explorer.gochain.io to see if it's there. 

Then setup your Ledger: https://support.ledger.com/hc/en-us/articles/360009766653-GoChain-GO-

**Comment by SPART20** [comment]
thanks so much for that info. I found the transaction on the Gochain network, have ledger setup. still not sure what info from the transaction I take to receive on the ledger, or even mew wallet (no receive on the wallet just send) sorry for my novice knowledge

________________________________
